internal/blocks/get.go

```diff
@@ -82,13 +82,31 @@ func get(
 	}
 
 	collections := make([]*flowsdk.Collection, 0)
+	collectionTxs := 0
 	if command.ContainsFlag(blockFlags.Include, "transactions") {
+		var lastCollection *flowsdk.Collection
 		for _, guarantee := range block.CollectionGuarantees {
 			collection, err := flow.GetCollection(context.Background(), guarantee.CollectionID)
 			if err != nil {
 				return nil, err
 			}
 			collections = append(collections, collection)
+			collectionTxs += len(collection.TransactionIDs)
+			lastCollection = collection
+		}
+
+		transactions, _, err := flow.GetTransactionsByBlockID(context.Background(), block.ID)
+		if err != nil {
+			return nil, err
+		}
+		// The last transaction returned from `flow.GetTransactionsByBlockID`,
+		// is the system chunk transaction. We add it as the last transaction
+		// in the last collection.
+		if lastCollection != nil && len(transactions) == (collectionTxs+1) {
```

Comment:
the SystemChunk transaction always have the same hash, that you can get from `blueprints.SystemChunkTransaction(chain).ID().String()` should we rely on ordering here or use that id?

Reply:
for mainnet it is f31815934bff124e332b3c8be5e1c7a949532707251a9f2f81def8cc9f3d1458 currently

Reply:
I recall that I had tried to use that one, however, the `transactions` here is a slice of `*flowsdk.Transaction`. While the `blueprints.SystemChunkTransaction(chain)` returns a `*flow.TransactionBody`.

Reply:
This is not true btw that is always has the same hash, it changes. :)